core/src/components/toast/test/basic/toast.e2e.ts

```diff
@@ -134,3 +134,31 @@ test.describe('toast: properties', () => {
     await expect(toast).toHaveClass(/my-custom-class/);
   });
 });
+
+test.describe('toast: duration config', () => {
+  test.beforeEach(({ skip }) => {
+    skip.rtl();
+    skip.mode('ios');
+  });
+  test('should have duration set to 0', async ({ page }) => {
+    await page.setContent(`
+    <ion-toast></ion-toast>
+  `);
+    const toast = page.locator('ion-toast');
+    await expect(toast).toHaveAttribute('duration', '0');
+  });
+
+  test('should have duration set to 5000', async ({ page }) => {
+    await page.setContent(`
+    <ion-toast></ion-toast>
+    <script>
+      window.Ionic = {
+        config: { toastDuration: 5000 } 
+      }
+    </script>
+  `);
+
+    const toast = page.locator('ion-toast');
+    await expect(toast).toHaveAttribute('duration', '5000');
```

Comment:
```suggestion
    await expect(toast).toHaveJSProperty('duration', 5000);
```

Sorry, I mistyped before. This needs to be `toHaveJSProperty` not `toHaveAttribute`